pkg/webui/console/views/gateway-overview/gateway-overview.js

```diff
@@ -120,6 +168,17 @@ export default class GatewayOverview extends React.Component {
             key: sharedMessages.frequencyPlan,
             value: frequency_plan_id ? <Tag content={frequency_plan_id} /> : undefined,
           },
+          {
+            key: m.globalConf,
+            value: (
+              <Button
```

Comment:
I think we should also hide this option when there is no `frequency_plan_id` set, otherwise you get an error:
```
{
  "code": 5,
  "message": "error:pkg/frequencyplans:not_found (frequency plan `` not found)",
  "details": [
    {
      "@type": "type.googleapis.com/ttn.lorawan.v3.ErrorDetails",
      "namespace": "pkg/frequencyplans",
      "name": "not_found",
      "message_format": "frequency plan `{id}` not found",
      "attributes": {
        "id": ""
      },
      "code": 5
    }
  ]
}
```